content/html/power-request-ci.html

```diff
@@ -0,0 +1,127 @@
+<div id="content">
+<!-- Formsender error script -->
+<script src="../../../theme/js/formsender-error.js"></script>
+  <div class="field field-name-body field-type-text-with-summary field-label-hidden">
+    <div class="field-items">
+      <div class="field-item even" property="content:encoded">
+        <p>Please use the form below to request access to the POWER CI service provided by the OSUOSL.</p> <p>This
+        access is intended only for <b><i>free and open source</i></b> projects who qualify and are approved by both
+        the OSUOSL and IBM. For proprietary sourced projects or products please use other POWER resources which can be
+        found <a href=https://developer.ibm.com/linuxonpower/cloud-resources/>here</a></p>
```

Comment:
Please add the following sentence at the end: The POWER CI service uses Docker to deploy Jenkins workers. In the future we will add access to use OpenStack to deploy Jenkins workers (ppc64le only).